TopFilms.py

```diff
@@ -0,0 +1,60 @@
+import connections as con
+
+title_rating_info = con.titles_info.join(con.rating_info,
+                con.titles_info.tconst == con.rating_info.tconst.alias('tconst'), 'inner') \
+        .withColumn('numVotes', con.rating_info.numVotes.cast('integer'))\
+        .drop(con.titles_info.tconst)
+
+
+def top_films_of_all_times_f():
```

Comment:
what is the purpose of `_f`?

Reply:
I did this to distinguish variables and functions

Reply:
Keyword 'def' isn't enough? :D